portfolio/src/main/java/com/google/sps/Data/User.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** A user and its data. */
+public final class User {
+
+  private final boolean loggedIn;
+  private final String logUrl;
```

Comment:
It's unclear to me what 'logUrl' is, might be worth picking a more clear variable name or commenting it!

Reply:
It's for either a log in URL or log out URL. I added a comment to the side, tell me if it's good or still needs change :)

Reply:
I would say the name 'logUrl' is a little confusing, because 'log' by itself makes me think of logging data, so I read this as a URL that is being logged. Maybe 'authenticationUrl', or 'logInOrOutUrl', or anything else that avoids this.

Reply:
Noted! I will go with authenticationURL.